examples/webgl_materials_alphahash.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - materials - alpha hash</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+	<body>
+		<!-- Import maps polyfill -->
+		<!-- Remove this when import maps will be widely supported -->
+		<script async src="https://unpkg.com/es-module-shims@1.6.3/dist/es-module-shims.js"></script>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js",
+					"three/addons/": "./jsm/"
+				}
+			}
+		</script>
+
+		<script type="module">
+
+			import * as THREE from 'three';
+
+			import Stats from 'three/addons/libs/stats.module.js';
+			import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
+			import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+
+			import { EffectComposer } from 'three/addons/postprocessing/EffectComposer.js';
+			import { RenderPass } from 'three/addons/postprocessing/RenderPass.js';
+			import { TAARenderPass } from 'three/addons/postprocessing/TAARenderPass.js';
+			import { OutputPass } from 'three/addons/postprocessing/OutputPass.js';
+
+			let camera, scene, renderer, controls, stats, mesh, material;
+
+			let composer, renderPass, taaRenderPass, outputPass;
+
+			let needsUpdate = false;
+
+			const amount = parseInt( window.location.search.slice( 1 ) ) || 3;
+			const count = Math.pow( amount, 3 );
+
+			const color = new THREE.Color();
+
+			const params = {
+				alpha: 0.5,
+				alphaHash: true,
+				taa: true,
+				sampleLevel: 2,
+			};
+
+			init();
+			animate();
+
+			function init() {
+
+				camera = new THREE.PerspectiveCamera( 60, window.innerWidth / window.innerHeight, 0.1, 100 );
+				camera.position.set( amount, amount, amount );
+				camera.lookAt( 0, 0, 0 );
+
+				scene = new THREE.Scene();
+
+				const light = new THREE.HemisphereLight( 0xffffff, 0x888888 );
+				light.position.set( 0, 1, 0 );
+				scene.add( light );
+
+				const geometry = new THREE.IcosahedronGeometry( 0.5, 3 );
+
+				material = new THREE.MeshStandardMaterial( {
```

Comment:
nit: `MeshStandardMaterial` requires an env map.

Consider double-sided, too.

Reply:
There are other examples that use `MeshStandardMaterial` without an env map, too. I see there no issue as long as `roughness`/`metalness` use their default values `1`/`0`.

Reply:
I've usually advised people "metallic materials require an env map" rather than all MeshStandardMaterial use... is there an issue with non-metallic materials too?

Let me test what this example looks like with THREE.RoomEnvironment, easy change.

Reply:
- https://github.com/mrdoob/three.js/pull/26344

Reply:
The specular reflectance of the material is non-zero and increases with metalness. This is consistent with the insulators and conductors the material is intended to model. The material is physically-based and requires an environment map for proper rendering -- not only for reflections, but also for proper ambient lighting.

Omitting the environment map is akin to omitting scene lighting.

Without an environment map, you get results like the screenshot above.